std_msgs/String rover_pose_channel
---

std_msgs/String rover_target_channel